app/controllers/ride_mechanics_controller.rb

```diff
@@ -0,0 +1,8 @@
+class RideMechanicsController < ApplicationController
+  def create
+    mechanic = Mechanic.find(params[:mechanic_id])
+    ride = Ride.find(params[:ride_id])
+    RideMechanic.create!(ride: ride, mechanic: mechanic)
+    redirect_to mechanic_path(mechanic.id)
+  end
```

Comment:
👍 

What you want to consider in the future is how many calls to the database you actually need to make. For lines 3 and 4, I would argue that you don't actually need to query the database at all and can just pass those IDs into line 5.

```ruby
def create
  mechanic_id = params[:mechanic_id]
  ride_id = params[:ride_id]
  RideMechanic.create!(ride_id: ride_id, mechanic_id: mechanic_id)
  redirect_to mechanic_path(mechanic_id)
end
```

This solution reduces 3 database calls to 1.